Selenium.Axe/Selenium.Axe/AxeRunOptions.cs

```diff
@@ -54,5 +70,43 @@ public class AxeRunOptions
         [JsonProperty("rules")]
         public Dictionary<string, RuleOptions> Rules { get; set; }
 
+        /// <summary>
+        /// Limit which result types are processed and aggregated. An approach you can take to reducing the time is use the resultTypes option. 
+        /// For eg, when set to [ResultTypes.Violations], scan results will only have the full details of the violations array and 
+        /// will only have one instance of each of the inapplicable, incomplete and pass arrays for each rule that has at least one of those entries. 
+        /// This will reduce the amount of computation that axe-core does for the unique selectors.
+        /// </summary>
+        [JsonProperty("resultTypes")]
+        public HashSet<ResultType> ResultTypes { get; set; }
+
+        /// <summary>
+        /// Returns xpath selectors for elements
+        /// </summary>
+        [JsonProperty("xpath")]
+        public bool? XPath { get; set; }
+
+        /// <summary>
+        /// Use absolute paths when creating element selectors
+        /// </summary>
+        [JsonProperty("absolutePaths")]
+        public bool? AbsolutePaths { get; set; }
+
+        /// <summary>
+        /// Tell axe to run inside iframes
+        /// </summary>
+        [JsonProperty("iframes")]
+        public bool? Iframes { get; set; }
+
+        /// <summary>
+        /// Scrolls elements back to the state before scan started
+        /// </summary>
+        [JsonProperty("restoreScroll")]
+        public bool? RestoreScroll { get; set; }
+
+        /// <summary>
+        /// How long (in milliseconds) axe waits for a response from embedded frames before timing out
+        /// </summary>
+        [JsonProperty("frameWaitTime")]
+        public int? FrameWaitTimeInMilliSec { get; set; }
```

Comment:
Half-abbreviating reads a little awkwardly. I think `FrameWaitTimeInMilliseconds` would be the most consistent with how the language's framework tends to spell/capitalize the word (see `TimeSpan.FromMilliseconds`)